Web/src/index.css

```diff
@@ -1,13 +1,12 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+:root {
```

Comment:
Neat! I didn't know that CSS supports variables natively.

No IE support tho lmao

![image](https://user-images.githubusercontent.com/11135744/66739166-af058d00-eea2-11e9-8786-89fbcb8d3b20.png)